wcmc_components/lib/wcmc_components/version.rb

```diff
@@ -1,3 +1,3 @@
 module WcmcComponents
-  VERSION = '0.2.0'
+  VERSION = '0.2.1'
```

Comment:
Sorry to stick ma nose in. Came to this space from looking at stuff in web-components channel and got curious.

Just wondered, should this be a minor version rather than a patch version increment?

Just that the feature you added to allow a column to refer to either an attribute or a method is like a new feature, rather than a fix?